source/process/docs-reviewers-guide.rst

```diff
@@ -0,0 +1,116 @@
+Reviewer's Guide for docs.mattermost.com 
+========================================
+
+Review of pull requests to Mattermost documentation passes through the following high-level steps: 
+
+1. Project Members and Approved Contributors submit PRs to the docs.mattermost.com repository. 
+2. Incoming PRs are assessed and assigned review requirements (e.g. technical PRs, PRs requiring Sphinx changes, etc.). 
+3. Reviewers with appropriate qualifications either approve and merge the changes, or leave feedback for the submitter to being the review cycle again. 
+4. Once merged, PRs are verified by a Project Member and the submitter's work acknowledged. 
+5. Reviewers with consistent performance are acknowledge with an increase in skill level. Reviewers with repeated inconsistency may have their level decreased (though this is rare). 
+
+The following sections lay out the details for this process: 
+
+Pre-reading 
+-------------
+
+- Please read `Contributor's Guide for docs.mattermost.com XXXXXX <XXXX>`_ for the process by which Approved Contributors submit PRs and engage with reviewers. 
+
+Review Labels 
```

Comment:
While I don't think the labels system is perfect, I think reviewers have different areas of expertise and levels of skill and aren't interchangeable. Some reviewers may be more technical, some have stronger grammar, and we need a system that delivers a process as the sum of our strengths. 

I think we should review the system, implement it as a pilot and evaluate the pros and cons vs. other options. 

I'm fine changing Style Guide to another name, as well as changing Needs ... Approval labels, etc. The goal is "sum of our strengths" in my mind. 

Editing would be after a merge--or maybe they're able to edit in a staging environment of some kind. I think the role is traditional editor